weekly_theme/src/weekly_theme/draw_theme.py

```diff
@@ -1,18 +1,17 @@
-import ipdb
-import numpy as np
+import random
 
-from config import DATA_DIR, THEME_FILE, PASSED_THEMES_FILE
+from .config import DATA_DIR, THEME_FILE, PASSED_THEMES_FILE
```

Comment:
I don't know this syntax: why do you need the dot before `config` for your imports?

Reply:
so I've clearly mishandled my imports but I'm not great at Python packages so I hacked the code until it worked...